ioos_qartod/qc_tests/qc.py

```diff
@@ -221,6 +221,45 @@ def flat_line_check(arr, low_reps, high_reps, eps, prev_qc=None):
     return flag_arr
 
 
+@add_qartod_ident(16, 'Time Series Flat Line Test')
+def ts_flat_line_check(arr, rep_cnt_suspect=3, rep_cnt_fail=5,
+                       eps=None, prev_qc=None):
+    """
+    Check for invariate observations and can be applied to all bulk wave
+    parameters.
+
+    """
+    if not eps:
+        raise ValueError("Must specify a tolerance value.")
+    if any([not isinstance(d, int) for d in [rep_cnt_suspect, rep_cnt_fail]]):
+        raise TypeError("Both suspect and fail repetitions must be type int")
+    flag_arr = np.ones_like(arr, dtype='uint8')
+    if rep_cnt_suspect >= rep_cnt_fail:
+        raise ValueError("Suspect reps must be less than fail reps")
+    it = np.nditer(arr)
+    # NOTE: Maybe Cython?  We can make it generic enough to be used in both
+    # flat line tests and reduce the code duplication.
```

Comment:
Awesome!  I don't "speak" c, but I am learning cython.

PS: This PR is just to start a conversation (see #56).  I will remove all those comments before merging.